src/main/resources/assets/changed/lang/zh_cn.json

```diff
@@ -552,5 +552,5 @@
   "changed.config.bpi.override_all_eye_styles": "覆盖所有眼睛样式以匹配",
   
   "fluid.changed.dark_latex": "液态黑胶",
-  "fluid.changed.white_latex": "液态白胶"
+  "fluid.changed.white_latex": "液态白胶",

```

Comment:
This was not an issue. This JSON will now fail to parse with the written comma. The last item in an array (or object) does not need a comma after it, as the comma indicates that there is another item.